it/controllers/IndexControllerISpec.scala

```diff
@@ -98,6 +143,63 @@ class IndexControllerISpec extends IntegrationSpecCommonBase {
     latePaymentPenalties = Some(Seq.empty[LatePaymentPenalty])
   )
 
+  val getPenaltyDetailsPayloadWithRemovedPoints = GetPenaltyDetails(
+    totalisations = None,
+    lateSubmissionPenalty = Some(LateSubmissionPenalty(
+      summary = LSPSummary(
+        activePenaltyPoints = 1,
+        regimeThreshold = 4,
+        inactivePenaltyPoints = 1,
```

Comment:
Ok, not sure when they are set to  inactivePenaltyPoints = 2 and activePenaltyPoints = 0, two tests fail with following errors (IndexControllerISpec.scala:1057, IndexControllerISpec.scala:1245):

"[No active late submission penalty points.]" was not equal to "[Your client has 1 penalty point. This is because:]"

Reply:
Checking this scenario now

Reply:
Done